workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/views/matplotlib.py

```diff
@@ -50,8 +53,104 @@ def __repr__(self):
 """
 )
 
+template_single_image_per_row = jinja2.Template(
+    """
+<h2> {{varfilter}} </h2>
+{% for varname in variables_to_plot %}
+<img src="{{ image_data[varname] }}" >
+{% endfor %}
+</table>
+"""
+)
+
+
+CBAR_RANGE = {
+    "eastward_wind_pressure_level_zonal_bias": 30,
+    "northward_wind_pressure_level_zonal_bias": 3,
+    "air_temperature_pressure_level_zonal_bias": 15,
+    "specific_humidity_pressure_level_zonal_bias": 1e-3,
+    "vertical_wind_pressure_level_zonal_bias": 0.02,
+}
+
+
+def _data_array_from_run_diags(run_diags, var):
+    values, run_coords = [], []
+    for run in run_diags.runs:
+        da = run_diags.get_variable(run, var)
+        if not np.isnan(da.mean(skipna=True)):
+            values.append(da)
+            run_coords.append(run)
+    return xr.concat(values, dim="run").assign_coords({"run": run_coords})
+
+
+def plot_2d_matplotlib_groupby_run(
```

Comment:
I just realized this is doing the opposite of groupby... I will change the name.